distribution/lib/Standard/Base/0.0.0-dev/src/Network/URI_With_Query.enso

```diff
@@ -94,11 +95,13 @@ type URI_With_Query
     host : Text | Nothing
     host self = self.uri.host
 
-    ## Get the authority (user info and host) part of this URI.
+    ## GROUP Metadata
+       Get the authority (user info and host) part of this URI.
     authority : Text | Nothing
     authority self = self.uri.authority
 
-    ## Get the port part of this URI.
+    ## GROUP Metadata
+       Get the port part of this URI.
```

Comment:
Like what @radeusgd said about XML elements, I wonder if this is something like `Selections`. Nearly every part of the URI is classified as `Metadata` here.